L1Trigger/L1TGlobal/src/AXOL1TLCondition.cc

```diff
@@ -129,9 +130,9 @@ const bool l1t::AXOL1TLCondition::evaluateCondition(const int bxEval) const {
   ap_fixed<18, 13> EtSumInput[EtSumVecSize];
 
   //declare result vectors +score
-  std::array<ap_fixed<10, 7>, 13> result;
+  std::array<ap_fixed<10, 7, AP_RND_CONV, AP_SAT>, 8> result;  //v3
   ap_ufixed<18, 14> loss;
-  std::pair<std::array<ap_fixed<10, 7>, 13>, ap_ufixed<18, 14>>
+  std::pair<std::array<ap_fixed<10, 7, AP_RND_CONV, AP_SAT>, 8>, ap_ufixed<18, 14>>
```

Comment:
Not going to be picky about this, but maybe having some `typedefs` for different model versions of input might also be useful.